pkg/build/nodeimage/internal/container/docker/pull.go

```diff
@@ -19,25 +19,29 @@ package docker
 import (
 	"time"
 
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
We would prefer to actually drop this dependency not add more dependency on it.

Pulling in dependencies staged from Kubernetes makes kind harder to embed. This is the last remaining one of those, which we currently only use for two small things.

Reply:
Thanks for the explanation！